text_generation/causal_lm/cpp/greedy_causal_lm.cpp

```diff
@@ -60,6 +60,9 @@ int main(int argc, char* argv[]) try {
     }
     // Compile models
     ov::Core core;
+
+    auto tokenizer_model = core.read_model(std::string{argv[1]} + "/openvino_tokenizer.xml");
+
     core.add_extension(OPENVINO_TOKENIZERS_PATH);  // OPENVINO_TOKENIZERS_PATH is defined in CMakeLists.txt
     // tokenizer and detokenizer work on CPU only
     ov::InferRequest tokenizer = core.compile_model(
```

Comment:
please move reading of tokenizer_model after add_extension just before compile_model, and do smth like that
```code
    ov::InferRequest tokenizer = core.compile_model(tokenizer_model, "CPU").create_infer_request();
```